elem_cfgr.go

```diff
@@ -15,10 +15,8 @@ type (
 	}
 )
 
-func (ecp *ElemCfgr) AttrInjection(key string) {
-	fragment := injection{
-		key: key,
-	}
+func (ecp *ElemCfgr) AttrsInjection(key string) {
```

Comment:
rename `ATTRIBUTE_INJECTION_NODE_KIND` to `ATTRIBUTES_INJECTION_NODE_KIND` and you will see all the  functions/methods that require some updates: `node.String()` and `node.New()`

Reply:
Done! But FYI, `String()` and `New()` are not methods of the `Node` structure. They are separate functions.